.github/workflows/build.yml

```diff
@@ -61,7 +61,7 @@ jobs:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
                   SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
               run: |
-                  ./.sonar/scanner/dotnet-sonarscanner begin /k:"COSC2650_Assignment" /o:"cosc2650" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.opencover.reportsPaths="**/coverage.opencover.xml" /d:sonar.coverage.exclusions="API/Program.cs","API/Startup.cs","API/Migrations/*"
+                  ./.sonar/scanner/dotnet-sonarscanner begin /k:"COSC2650_Assignment" /o:"cosc2650" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.opencover.reportsPaths="**/coverage.opencover.xml" /d:sonar.coverage.exclusions="API/Program.cs","API/Startup.cs","API/Migrations/*","API/Models/*","API/GraphQL/Users/*"
```

Comment:
Should we be ignoring these two folders for code coverage?

Reply:
I thought we discussed not testing the GraphQL stuff as long as we are testing the Services they use?

In terms of models, I thought it was OTT considering you would just have to Create the object to test it correct?

These are intended as questions and am happy to take a different approach, what do you think?